product_docs/docs/pgd/5/routing/proxy.mdx

```diff
@@ -54,6 +54,23 @@ PGD CLI provides a few commands to manage proxies in a PGD cluster, such as `cre
 
 See [Connection management](../routing) for more information on the PGD side of configuration and management of PGD Proxy.
 
+### Proxy health check
+
+PGD Proxy provides the following HTTP(s) health check API endpoints. These APIs are `GET` requests and do not require a request body. See [Configurations](installing_proxy#configuring-health-check) for more details about enabling and configuring this.
+
+```
+GET /health/is-ready
+GET /health/is-live
+```
+
+#### Readiness
+
+On receiving valid request, proxy checks if it can successfully route connections to the current write leader. If the check returns successfully then API returns `true` with HTTP status code `200 (OK)` else `false` with HTTP status code `500 (Internal Server Error)`.
+
+#### Liveness
+
+Liveness check either return `true` with HTTP status code `200 (OK)` or an error but never `false`. This is because the HTTP server listening for request is automatically stopped if the Proxy service fails to start or exists.
```

Comment:
Don't you mean the request only ever returns 'true'; if the request cannot be made, it is because the http server has been stopped because the Proxy service has not started or stopped. 

"fails to start or exists" is contradictory.

Reply:
Yes, that's what it means. I used the word "**exit**"  because Proxy can be stopped gracefully or exit abruptly due to some critical error conditions that cannot be handled by the proxy

Reply:
Ah, exit.... So should be "fails to start or exits" (not exists).